digiwf-integrations/digiwf-ticket-integration/digiwf-ticket-integration-core/src/test/java/de/muenchen/oss/digiwf/ticket/integration/adapter/in/WriteArticleMessageProcessorTest.java

```diff
@@ -0,0 +1,82 @@
+package de.muenchen.oss.digiwf.ticket.integration.adapter.in;
+
+import de.muenchen.oss.digiwf.message.process.api.ErrorApi;
+import de.muenchen.oss.digiwf.message.process.api.ProcessApi;
+import de.muenchen.oss.digiwf.ticket.integration.adapter.in.streaming.TicketMessageProcessor;
+import de.muenchen.oss.digiwf.ticket.integration.adapter.in.streaming.WriteArticleDto;
+import de.muenchen.oss.digiwf.ticket.integration.application.port.in.WriteArticleInPort;
+import de.muenchen.oss.digiwf.ticket.integration.domain.model.Article;
+import de.muenchen.oss.digiwf.ticket.integration.domain.model.TicketStatus;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+
+import java.util.Map;
+
+import static de.muenchen.oss.digiwf.message.common.MessageConstants.*;
+import static org.mockito.ArgumentMatchers.any;
+
+class WriteArticleMessageProcessorTest {
+
+    private WriteArticleInPort writeArticleInPort = Mockito.mock(WriteArticleInPort.class);
+
+    private ProcessApi processApi = Mockito.mock(ProcessApi.class);
+
+    private ErrorApi errorApi = Mockito.mock(ErrorApi.class);
+
+    private TicketMessageProcessor messageProcessor = new TicketMessageProcessor(
+            writeArticleInPort,
+            processApi,
+            errorApi);
+
+    @BeforeEach
+    void setup() {
+
+        Mockito.doNothing().when(writeArticleInPort).writeArticle(any(), any(), any());
+
```

Comment:
```suggestion
        Mockito.doNothing().when(writeArticleInPort).writeArticle(any(), any(), any());
```
Es sind ein bisschen viele Whitespaces in dem Test